src/lib/openapi/spec/feature-tag-schema.ts

```diff
@@ -3,23 +3,36 @@ import { FromSchema } from 'json-schema-to-ts';
 export const featureTagSchema = {
     $id: '#/components/schemas/featureTagSchema',
     type: 'object',
+    description: 'Describes a tag applied to a feature',
     additionalProperties: false,
     required: ['featureName', 'tagValue'],
     properties: {
         featureName: {
             type: 'string',
+            example: 'my-feature',
+            description: 'The name of the feature this tag is applied to',
         },
         tagType: {
             type: 'string',
+            example: 'simple',
+            description: 'The type of tag',
         },
         tagValue: {
             type: 'string',
+            example: 'my-tag',
+            description: 'The value of the tag',
         },
         type: {
+            deprecated: true,
             type: 'string',
+            description:
+                'This field is deprecated and currently unused, use tagType instead',
```

Comment:
Wait: this is a response schema, right? 

If so, maybe something like this?

```suggestion
            description:
                'The [type](https://docs.getunleash.io/reference/tags#tag-types tag types) of the tag. This property is deprecated and will be removed in a future version of Unleash. Superseded by the `tagType` property.',,
            example: 'simple',
```

Reply:
Sure, I believe this was an oversight on the past, I couldn't find where this is being used and it's currently returned as `undefined` (i.e. field is not present in the response). I tested the APIs and none of the APIs using this schema returned these fields